r4/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.r4"
+version = "3.0.1"
```

Comment:
This should be a major version bump.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/dcc9e9fabd3f6b692a6a4768f841877eba6461ae